### ISSUE TITLE
Jelly Library Fails to Parse Cross-File Modules

### ISSUE THREAD

**Issue by willowwy** [cs-au-dk/jelly#15]
I attempted to implement Example1 from the paper associated with the Jelly library. This involved creating two separate files and importing modules from one file to another. Despite following the documentation's guidelines closely, Jelly is unable to parse the cross-file modules, resulting in an import failure. 
![image](https://github.com/cs-au-dk/jelly/assets/76858197/24ec4d9c-0994-4865-b393-251a4c314fd9)

### Actual Behavior
An error occurs during the parsing process.
**Error: Unable to resolve module**
![image](https://github.com/cs-au-dk/jelly/assets/76858197/0d730c3c-c140-43a7-80be-d918b5d59027)

I have attempted clearing the npm cache and reinstalling node_modules, but the issue persists.
Node.js version: v20.0.0
NPM version:9.6.4

**Comment by amoeller** [comment]
You set `-b .` (sets the base directory to .) but analyze files in another directory. Try removing `-b .` or replacing `.` by `D:\Study\Lab_NPM\try_jelly\test`.

**Comment by willowwy** [comment]
I have retried your suggested solutions, but I'm still encountering the same issue as before, unable to resolve it.
![image](https://github.com/cs-au-dk/jelly/assets/76858197/5e2bff78-80c9-4791-a20f-c48f1b1f6e00)


I've made multiple attempts, exploring all possible avenues, yet the problem persists.

```bash
node ../jelly/lib/main.js -j cg.json -b . D:\Study\Lab_NPM\try_jelly\test
node ../jelly/lib/main.js -j cg.json -b D:\Study\Lab_NPM\try_jelly\test D:\Study\Lab_NPM\try_jelly\test
node ../jelly/lib/main.js -j cg.json -b D:\Study\Lab_NPM\try_jelly\test .
node ../jelly/lib/main.js -j cg.json D:\Study\Lab_NPM\try_jelly\test
node ../jelly/lib/main.js -j cg.json .

jelly -j cg.json -b . D:\Study\Lab_NPM\try_jelly\test
jelly -j cg.json -b D:\Study\Lab_NPM\try_jelly\test D:\Study\Lab_NPM\try_jelly\test
jelly -j cg.json -b D:\Study\Lab_NPM\try_jelly\test .
jelly -j cg.json D:\Study\Lab_NPM\try_jelly\test
jelly -j cg.json .
```
Additionally, I've provided the structure of my files for a clearer understanding of the setup:

```
try_jelly
├─ jelly
│  └─ ……
└─ test
   ├─ cg.html
   ├─ cg.json
   ├─ clinet1.js
   ├─ lib1.js
   └─ package.json
```

**Comment by willowwy** [comment]
It turned out to be an issue with how Linux and Windows handle paths differently. After switching to Linux, everything runs smoothly now. Thanks a lot!

**Comment by amoeller** [comment]
Should be fixed by c197c6b.
